src/hooks/twilio.ts

```diff
@@ -92,12 +92,12 @@ export const useParticipantState = ({
     };
   }, [participant]);
 
-  // Mute/unmute audio
-  const [isMuted, setMuted] = useState<boolean>(defaultMute);
-  const toggleMuted = useCallback(
-    () => setMuted((prevIsMuted) => !prevIsMuted),
-    []
-  );
+  const {
+    isShown: isMuted,
+
+    setShown: setMuted,
+    toggle: toggleMuted,
+  } = useShowHide(defaultMute);
```

Comment:
Why did we remove the `// Mute/unmute audio` comment here? I believe when I called this out in a previous thread you said it was restored in the latest commit, but seemingly that's not true? (or the latest code isn't pushed)

Reply:
returned comment in https://github.com/sparkletown/sparkle/pull/1299/commits/ce562c6a4616118ea904bb8f088ca751787a5f95